project/Dependencies.scala

```diff
@@ -59,8 +59,6 @@ object Dependencies {
       exclude("io.netty", "netty-common"),
 
     excludeGuava("com.google.apis"     % "google-api-services-storage"      % "v1-rev20190910-1.30.3"),
-    excludeGuava("com.google.apis"     % "google-api-services-sheets"       % "v4-rev20191001-1.30.3"),
-    excludeGuava("com.google.apis"     % "google-api-services-cloudbilling" % "v1-rev20191005-1.30.3"),
```

Comment:
neither of  these is used, with the exception of the orphaned imports and references below